### PR TITLE
Improve browsers icons

### LINKED ISSUE #317: Improve browsers icons
Now we are doing:

```js
    iconElem.src = `/assets/browser_logos/${id}.svg`;
```

It has 2 problems:
- When Can I Use added new browser, we will have 404 error
- SVG files are not processed by Vite. For instance we do not add cache busters.

Instead we should import all icons by `import imgUrl from './img.svg'`

### PR DESCRIPTION
**Pull request by a-chabin**
Closes #317

### CODE REVIEW

File: client/view/BrowserStats/browserStats.js [hunk 1]
```diff
@@ -86,7 +87,7 @@ export function updateBrowsersStats(data) {
     iconCell.classList.add('BrowsersStat__td')
     let iconElem = document.createElement('img')
     iconElem.classList.add('BrowsersStat__icon')
-    iconElem.src = `/assets/browser_logos/${id}.svg`
+    iconElem.src = browsersIcons[id]
```
Comment (ai) [inline client/view/BrowserStats/browserStats.js:90]: We need to take care of the case when we do not have icon for some browser.

Can I Use sometimes adds browsers.

File: client/data/browsers-icons.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable camelcase */
+export { default as and_chr } from '../assets/browser_logos/and_chr.svg'
```
Comment (ai) [inline client/data/browsers-icons.js:2]: Can we move icons to the folder in `data`?

File: client/view/BrowserStats/browserStats.js (hunk 1 above)
Comment (a-chabin) [reply client/view/BrowserStats/browserStats.js:90]: Added the same TODO for wikipedia

**Comment by a-chabin** [comment]
Increase size-limit to 30 kb 🥶

**Pull request merged**